src/faq.md

```diff
@@ -144,27 +148,41 @@ appreciation through careful design choices and fair communication about them.
 
 ### Q. Is Dart a statically typed language?
 
-Dart is statically typed if you opt in to [strong mode]
-and use a compiler (such as the Dart Development Compiler ([DDC])),
+Dart 1.x is statically typed if you opt in to [strong mode]
+and use a compiler (such as the Dart Development Compiler
+([dartdevc]({{site.webdev}}/tools/dartdevc))),
 that has both static and runtime checks. In this environment, Dart
```

Comment:
"this environment" is unclear. "dartdevc" might be clearer, but perhaps not what you want to say.